understanding/21/label-in-name.html

```diff
@@ -9,22 +9,21 @@
+			<h3>Identifying label text</h3>
+<p>The label for a component is usually a text string in close proximity to the component. The typical positioning for left to right languages is:
 <ul><li>immediately to the left of comboboxes, dropdown lists, text inputs and other widgets (or in the absence of left-side labels, immediately above and aligned with the left edge of each input)</li>
 <li>immediately to the right of checkboxes and radio buttons</li>
 <li>inside buttons and tabs or immediately below icons serving as buttons</li>
 </ul>
-The rationale for some of these conventions is explained in <a href="https://www.w3.org/WAI/GL/2016/WD-WCAG20-TECHS-20160105/G162">G162: Positioning labels to maximize predictability of relationships</a>.
+The rationale for some of these conventions is explained in <a href="https://www.w3.org/TR/WCAG-TECHS/G162.html">G162: Positioning labels to maximize predictability of relationships</a>.
 </p>
-<p>Isolating the label to the single string in close proximity to the component makes it easier for developers, testers and end users to identify the label targeted for evaluation in this SC. Obvious labeling allows users of speech recognition technologies to interact with the element via its conventionally positioned label, and allows users of screen reading technologies to enjoy consistency between the visible label and the announced name of the component.</p>
-<p>Note that placeholder text within an input field is not considered an appropriate means of providing a label. The <a href="https://www.w3.org/TR/html52/sec-forms.html#the-placeholder-attribute">HTML5 specification</a> states <q>The placeholder attribute should not be used as an alternative to a &lt;label&gt;.</q> However, it is worth noting that "label" in that HTML5 statement is in code brackets and links to the <code>label</code> element. For the purposes of this Label in Name Success Criterion, "label" is not used in such a programmatic sense but is simply referring to a text string in close visual proximity to a component. As such, in the absence of any other nearby text string (as described in the preceding list), if an input contains placeholder text, such text may be a candidate for Label in Name. This is supported both through the accessible name calculation (discussed later) and from the practical sense that where a visible label is not otherwise provided, it is likely that a speech-input user may attempt to use the placeholder text value as a means of interacting with the input.</p>
+<p>When labels use a single string of text in close proximity to the component, developers, testers, and end users find it easy to identify the label targeted for evaluation in this SC. Obvious labeling allows users of speech recognition technologies to interact with the element via its conventionally positioned label, and allows users of screen reading technologies to enjoy consistency between the visible label and the announced name of the component.</p>
```

Comment:
maybe slight change for readability
"If labels use a single string of text in close proximity to the component, then developers..."
or better
"If labels are interpreted as a single..."

Concern here is that it now looks like we're offering advice on labelling, whereas before it was advice on how to interpret the label. Do you see the difference? The former is design advice, the latter is developer/tester interpretation guidance, which is I believe what this SC is about. We're not trying to alter how designers label stuff. Just ensure what they've done is reflected programmatically for the betterment of speech users.